fetch_submissions.py

```diff
@@ -141,13 +145,22 @@ def save_submission_as_file(submission):
 
 
 if __name__ == '__main__':
-    if (len(sys.argv) != 2):
-        print('Usage: python3 fetch_submissions.py <username>')
+    parser = argparse.ArgumentParser(description="A simple script to download your fastest, accepted solutions from Leetcode!")
```

Comment:
I do not think that we need argparse. The script is simple enough to not need command line arguments for now